.github/workflows/nightly.yml

```diff
@@ -14,13 +14,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rocm-version: ["6.4.1", "7.0"]
+        rocm-version: ["6.4.1", "7.0.0"]
         include:
           - rocm-version: "6.4.1"
             runner-label: "mi-250"
-          - rocm-version: "7.0"
-            rocm-build-job: "compute-rocm-dkms-no-npi-hipclang"
-            rocm-build-num: "16322"
+          - rocm-version: "7.0.0"
+            rocm-build-job: "compute-rocm-rel-7.0"
+            rocm-build-num: "40"
```

Comment:
These should be parameterized in the master branch so that we dont need commits to produce the artifact we want! However, that can be a separate PR

Reply:
Yes, the parameterized inputs and also public URL changes will be done in master in a separate PR after the release is done